docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-offer-feature-integration.md

```diff
@@ -1383,6 +1391,8 @@ merchant_product_offer.sold_by,Sold by,en_US
 merchant_product_offer.sold_by,Verkauft durch,de_DE
 merchant.sold_by,Sold by,en_US
 merchant.sold_by,Verkauft durch,de_DE
+merchant_product.sold_by,Sold by,en_US
```

Comment:
This one translationes seems should not be hear, cos used in MerchantProductWidget module, so need to be the part of Marketplace Product feature IG

Reply:
please move to the right place